main.rb

```diff
@@ -0,0 +1,76 @@
+require_relative 'app'
+
+def run_option
+  puts 'Welcome to the Library App!'
+  puts 'Please choose an option:'
+  puts '1 - List all books.'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
+  puts '6 - List all rentals for a given person ID'
+  puts '7 - Quit'
+end
+
+def create_person(app)
+  puts 'Do you want to create a student(1) or a teacher(2)? [Input the number]:-'
+  option = gets.chomp
+  parent_permission = true
+  puts 'age:-'
+  age = gets.chomp.to_i
+  puts 'name:-'
+  name = gets.chomp
+  puts 'Has parent permission?[Y/N]'
+  val = gets.chomp
+  parent_permission = false if %w[N n NO no No].include?(val)
+
+  case option.to_i
+  when 1
+    app.create_student(age, name, parent_permission)
+  when 2
+    puts 'specialization:-'
+    specialization = gets.chomp
+    app.create_teacher(age, specialization, name, parent_permission)
+  else
+    puts 'Invalid input ):'
+  end
+end
```

Comment:
Example video
![image](https://github.com/Mikey1-nuel/school_library/assets/78251927/42602977-1063-4e2a-9b0d-334bb2d5e285)

Your app console
![image](https://github.com/Mikey1-nuel/school_library/assets/78251927/dcae89a8-49de-4063-83c7-0a17f5711308)

According to the project requirements, our console app should behave in the same way as it is in the example. 🤗

Please implement the following to make your app behave in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)

- 👀 `Has parent permission?` is not required to create a `Teacher`. 🤓 please remove it 👍
- Kindly add a success message that will be displayed when a new item is created. e.g `Person created successfully`